pkg/controller/disruption/disruption.go

```diff
@@ -260,15 +256,11 @@ func (dc *DisruptionController) getPodDeployment(pod *v1.Pod) (*controllerAndSca
 	return &controllerAndScale{deployment.UID, *(deployment.Spec.Replicas)}, nil
 }
 
-func (dc *DisruptionController) getPodReplicationController(pod *v1.Pod) (*controllerAndScale, error) {
-	controllerRef := metav1.GetControllerOf(pod)
-	if controllerRef == nil {
-		return nil, nil
-	}
+func (dc *DisruptionController) getPodReplicationController(controllerRef *metav1.OwnerReference, namespace string) (*controllerAndScale, error) {
 	if controllerRef.Kind != controllerKindRC.Kind {
```

Comment:
need to also make sure `schema.ParseGroupVersion(controllerRef.APIVersion).Group == ""`